packages/core/package.json

```diff
@@ -26,10 +26,13 @@
   "scripts": {
     "build": "rm -rf ./build && yarn tsc",
     "unit-tests": "jest \".*unit.spec.ts\"",
-    "integration-tests": "jest --runInBand \".*integration.spec.ts\""
+    "integration-tests": "jest --runInBand \".*integration.spec.ts\"",
+    "lint": "eslint src --ignore-path ../../.gitignore --max-warnings 0 --ext .js,.ts",
+    "lint-fix": "yarn lint --fix"
```

Comment:
Should we also add a `lint` and `lint-fix` script to the root `package.json` file? So that we can run `yarn lint` from the root and that gets executed on all supported packages?

Reply:
The `lint` script is already present. I will add the `lint-fix`